src/cider/exec/operator/join/CiderStdUnorderedHashTable.h

```diff
@@ -18,32 +18,31 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-#include <folly/FBVector.h>
 
 // used as test
 template <typename Key, typename Value>
 class StdMapDuplicateKeyWrapper {
  public:
   StdMapDuplicateKeyWrapper(){};
-  folly::fbvector<Value> find(Key key) {
+  std::vector<Value> findAll(Key key) {
     auto iter = umap_.find(key);
     if (iter != umap_.end()) {
       return iter->second;
     }
-    folly::fbvector<Value> empty_res;
+    std::vector<Value> empty_res;
```

Comment:
So we switch to use `std::vector` instead of folly one, any background and will there be any performance issue?

Reply:
This is based on the feedback from infra team. Folly is a large lib and use anything from folly in codegen context is inappropriate. And std::vector is lighter, so switch to std::vector